prog/Summarize_building_permits.sas

```diff
@@ -0,0 +1,71 @@
+/**************************************************************************
+ Program:  Summarize_building_permits.sas
+ Library:  DCRA
+ Project:  NeighborhoodInfo DC
+ Author:   Rob Pitingolo
+ Created:  03/19/19
+ Version:  SAS 9.4
+ Environment:  Windows
+ 
+ Description:  Summarize building permits by geography.
+
+ Modifications:
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( DCRA )
+%DCData_lib( OCTO )
+
+%let start_yr = 2009;
+%let end_yr = 2018;
+
+%let summaryvars = permit_&year. permit_construction_&year. permits_homeoccupation_&year. permits_postcard_&year. permits_shopdrawing_&year. permits_supplemental_&year.;
+
+%macro permits_by_geo (geo);
+
+proc summary data= permits_&year._geo;
```

Comment:
This isn't right because it's not referencing the permanent data set. 

It might be better to define a second issue for creating the summary files.